.github/workflows/image-build-pdf-service.yaml

```
@@ -67,4 +67,4 @@ jobs:
         run: |
           docker build  -t pucarsolutionsdev.azurecr.io/pdf-service-db:${{ env.VERSION }}-${{ env.commit_hash }} -f  Dockerfile .
           docker push pucarsolutionsdev.azurecr.io/pdf-service-db:${{ env.VERSION }}-${{ env.commit_hash }}
-        working-directory: common/pdf-service/migration
+        working-directory: common/pdf-service/migration
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)